deps/rabbitmq_shovel/src/rabbit_local_shovel.erl

```diff
@@ -425,8 +457,11 @@ add_routing(Msg0, Dest) ->
         RK -> mc:set_annotation(?ANN_ROUTING_KEYS, [RK], Msg)
     end.
 
-status(_) ->
-    running.
+status(State) ->
+    case is_blocked(State) of
+        true -> blocked;
+        false -> running
+    end.
 
 pending_count(_State) ->
     0.
```

Comment:
now that there can be pending messages in local shovels too, pending_count should be the length of `pending_delivery`